README.rst

```diff
@@ -14,23 +14,122 @@ Lasagne
 =======
 
 Lasagne is a lightweight library to build and train neural networks in Theano.
+Its main features are:
 
-For support, please refer to the lasagne-users mailing list: https://groups.google.com/forum/#!forum/lasagne-users
+* Supports feed-forward networks such as Convolutional Neural Networks (CNNs),
+  recurrent networks including Long Short-Term Memory (LSTM) [coming soon!],
+  and any combination thereof
+* Allows architectures of multiple inputs and multiple outputs, including
+  auxiliary classifiers
+* Many optimization methods including Nesterov momentum, RMSprop and ADAM
+* Freely definable cost function and no need to derive gradients due to
+  Theano's symbolic differentiation
+* Transparent support of CPUs and GPUs due to Theano's expression compiler
 
-Documentation: http://lasagne.readthedocs.org/ (work in progress)
+Its design is governed by `six principles
+<http://lasagne.readthedocs.org/en/latest/user/development.html#philosophy>`_:
 
-Lasagne is a work in progress, input is welcome.
+* **Simplicity**: Be easy to use and easy to extend, with as few abstractions
+  as possible, to facilitate research
+* **Small interfaces**: Have as few classes and methods as possible
+* **Don't get in the way**: Do not obstruct users with features they decide not
+  to use, and allow all parts to be used independently
+* **Transparency**: Do not hide Theano behind abstractions, directly process and
+  return Theano expressions or Python / numpy data types
+* **Focus**: "Do one thing and do it well"
```

Comment:
I've removed talking about feedforward neural networks here in the shortened version. I'm thinking more along the lines of not aiming to provide a solution for everything involved in deep learning, but only facilitating some steps that would be more difficult in vanilla Theano (for now: creating a forward expression along with its learnable parameters, and creating update dictionaries).